Makefile

```diff
@@ -126,7 +126,11 @@ unit: btcd
 
 unit-cover: $(GOACC_BIN)
 	@$(call print, "Running unit coverage tests.")
-	$(GOACC_BIN) $$(go list ./... | grep -v lnrpc) -- -test.tags="$(DEV_TAGS) $(LOG_TAGS)"
+	if [ ! "$(pkg)" = "" ]; then \
```

Comment:
this definitely works, tho i have a simpler fix in mind :)

most of the conditional flag parsing is done `make/testing_flags.mk`. here's an example of one that uses a default if none is provided https://github.com/lightningnetwork/lnd/blob/master/make/testing_flags.mk#L33

what do you think of defining a `COVER_PKG` variable there, with a default of `./...`, and using that as the argument to `go list`?

Reply:
fwiw, the argument to go list is absolute, so we'd want `COVER_PKG := $(PKG)/pkg` when an arg is provided, which will prepend the full lnd package name

Reply:
actually, scratch the last part, `./...` is obviously relative, so we can just prepend a `./` 😅

Reply:
Yes, understood, this makes sense. Fixed it up will push here in a mo'